pkg/k8sutil/create.go

```diff
@@ -28,8 +28,11 @@ import (
 	v1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	yamlserializer "k8s.io/apimachinery/pkg/runtime/serializer/yaml"
 	"k8s.io/apimachinery/pkg/util/yaml"
 	corev1typed "k8s.io/client-go/kubernetes/typed/core/v1"
+	sigyaml "sigs.k8s.io/yaml"
```

Comment:
We always import it as just `yaml`, so no need for `sigyaml`.

Reply:
There is already a lib imported that is used as `yaml`. So it was only preferable to not introduce unnecessary changes in other places.

Reply:
Basically this change is trying to consistent with at the file level than at the project level.

Reply:
I'd be consistent on a project level, as `sigs.k8s.io/yaml` is used more frequently than `k8s.io/apimachinery/pkg/util/yaml`, so it can have a named import.

Reply:
Alright then I will add a new commit to separate the changes.

Reply:
What separate commit? Just import path should changed in newly added code?

Reply:
It cannot be done because the existing `yaml` import is used in existing code. So in the additional code I imported `sigs.k8s.io/yaml` as `sigyaml`.

As you suggest I have made changes in the latest commit PTAL.

Reply:
Oh, there are 3 yaml packages in this file, I didn't notice, oh my... :smile:

Reply:
@surajssd can we change the import before we add new code to make reviewing easier?

Reply:
Can we pass on this one? I tried to move it on top but it is creating weird merge conflicts.

Reply:
Do the following:
- `git rebase -i master`
- Add `b` at the top.
- Commit your change for `k8s.io/apimachinery/pkg/util/yaml` package.
- `git rebase --continue` and fix conflict on new imports (simple one).
- Do `git rebase -i master` again and fixup your last commit with one adding code to `pkg/k8sutil/create.go` file.